tket2-hseries/Cargo.toml

```
@@ -0,0 +1,33 @@
+[package]
+name = "tket2-hseries"
+version = "0.1.0"
+edition.workspace = true
+rust-version.workspace = true
+
+license.workspace = true
+readme = "README.md"
+documentation = "https://docs.rs/tket2-hseries"
+homepage.workspace = true
+repository.workspace = true
+description = "TKET2 tool for preparing and validating `Hugr`s for compilation targeting Quantinuum H-series  quantum computers"
+keywords = ["Quantum", "Quantinuum"]
+categories = ["compilers"]
+
+[features]
+default = ["cli"]
+cli = ["dep:hugr-cli", "dep:clap"]
+
+[dependencies]
+hugr.workspace = true
+hugr-cli = { workspace = true, optional = true }
+clap = { workspace = true, optional = true, features = ["derive"] }
+tket2.workspace = true
```

`tket2` is currently a workspace dependency, but I think it works better with `release-plz` if you define explicitly here;
```suggestion
tket2 = { path = "../tket2", version = "0.1.0-alpha.1" }
```
That version will get bumped in the release PR.

Can you also update tket2-py, and remove it from the workspace?